CHANGES.rst

```diff
@@ -149,6 +149,8 @@ Bug fixes
 
 - ``astropy.vo``
 
+  - Fixed spurious failure for Cone Search prediction test. [#4382]
```

Comment:
Maybe something like "Relaxed expected accuracy of Cone Search prediction test"?  That is, it's not really clear this is a "fix".